internal/context/init.go

```diff
@@ -106,8 +103,9 @@ func loadAllModules(moduleSources []string) map[string]moduleconfig.ModuleConfig
 			exit.Fatal("Unable to load module:  %v\n", err)
 		}
 		modules[mod.Name] = mod
+		mappedSources[mod.Name] = moduleSource
 	}
-	return modules
+	return modules, mappedSources
```

Comment:
is it bad to just add source to the moduleConfig itself?

Reply:
ModuleConfig represents the structure of the module file, so we don't want it in there.